source/braille.py

```diff
@@ -1100,6 +1123,9 @@ def regionPosToBufferPos(self, region, pos, allowNearest=False):
 			return start
 		raise LookupError("No such position")
 
+	def bufferPositionsToRawText(self, startPos, endPos):
+		return self.rawText[self.brailleToRawPos[startPos]:self.brailleToRawPos[endPos-1]+1]

```

Comment:
Does this call `_get_brailleToRawPos`? If so, that's a double call that can be avoided.

Reply:
This is a good point indeed. May be we should cache these properties by default (i.e. self._cache_brailleToRawPos=True).

Reply:
That, or in this specific case you could make it a local variable.

Reply:
... And if we cache it, then the opposite getter should be cached too I think.